default/stringtables/en.txt

```diff
@@ -8885,7 +8894,7 @@ TEMPORAL_ANOMALY_SPECIAL
 Temporal Anomaly
 
 TEMPORAL_ANOMALY_SPECIAL_DESC
-'''Increases [[metertype METER_TARGET_RESEARCH]] on this planet by 5 per [[metertype METER_POPULATION]] when Focus is set to [[metertype METER_RESEARCH]]. Decreases [[metertype METER_TARGET_POPULATION]] (regardless of Focus) depending on the planet size:
+'''Increases [[metertype METER_TARGET_RESEARCH]] on this planet by [[value TEMPORAL_ANOMALY_TARGET_RESEARCH_PERPOP]] per [[metertype METER_POPULATION]] when Focus is set to [[metertype METER_RESEARCH]]. Decreases [[metertype METER_TARGET_POPULATION]] (regardless of Focus) by planet size per [[TEMPORAL_ANOMALY_TARGET_POPULATION_PERSIZE]]:
 • [[SZ_TINY]] (-5)
```

Comment:
This won't update if TEMPORAL_ANOMALY_TARGET_POPULATION_PERSIZE named value changes, although the previous line allows the reader to infer the real values (if planet sizes are known).
I've considered removing this bullet points and linking to the values of the planet sizes (not sure if there is a Pedia page for that already or not), or adding five new named values, one per planet size (and more new named values for other stringtable entries like this one with per planet size modifiers. Thoughts?

Reply:
`[[TEMPORAL_ANOMALY_TARGET_POPULATION_PERSIZE]]` is a stringtable lookup macro, isn't it?

Reply:
> `[[TEMPORAL_ANOMALY_TARGET_POPULATION_PERSIZE]]` is a stringtable lookup macro, isn't it?

I don't think so, but I don't have the concepts clear.
AFAIK, it is a NamedReal defined in https://github.com/freeorion/freeorion/blob/0ce3083f5f5b351f2667af7a48c7d76a2332a4a5/default/scripting/specials/planet/TEMPORAL_ANOMOLY.focs.txt#L20

For some reason that @agrrr3 might understand (I don't), it is not enough to declare the named value in the FOCS file: each named value used in the stringtables must have a corresponding stringtable entry (macro?) with a text that will show in the tootips over the stringtable calls to `[[value <named value name>]]` and in the [[Test - NamedValues]] Pedia page.

Reply:
There might be a "TEMPORAL_ANOMALY_TARGET_POPULATION_PERSIZE" named valueref, but here you're just doing a stringtable substitution macro as far as I can tell. There is no `value` in `[[TEMPORAL_ANOMALY_TARGET_POPULATION_PERSIZE]]`.

Reply:
The line you see is before I fixed it with the latest force-push. Current state of the PR has that missing value fixed:
https://github.com/freeorion/freeorion/blob/0ce3083f5f5b351f2667af7a48c7d76a2332a4a5/default/stringtables/en.txt#L8897

Reply:
OK.

Regarding the original question, a proper phrasing in the description before the list of planet sizes should be adequate. Could add a bunch more value-refs, but they'd also be a maintainance issue if not actually used in the FOCS that generates the relevant effects in-game.